tests/test_downloader_handlers.py

```diff
@@ -108,14 +108,13 @@ def test_lazy_handlers(self):
 class FileTestCase(unittest.TestCase):
 
     def setUp(self):
-        self.tmpname = self.mktemp()
-        with open(self.tmpname + '^', 'w') as f:
-            f.write('0123456789')
+        self.tmpname = Path(self.mktemp() + '^')
```

Comment:
Note to self: find (again) why is this ^ needed in this test suite and add a comment. Or remove it if it's not actually needed.

Reply:
#5736